Microsoft.NET.Build.Containers.IntegrationTests/TestSettings.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+
+namespace Microsoft.NET.Build.Containers.IntegrationTests;
+
+internal static class TestSettings
```

Comment:
I opted in not to clean up tests for time being. This goes to temp and will be eventually cleaned up.
After move to sdk we may consider to reuse part of their infrastructure or switch to using (disposable) fixtures for this purpose.